Cygwin specific: on Linux, g_module_build_path assumes that if the "lib"
prefix is present, that the full name has been given and won't add it again.
On Cygwin, the "cyg" prefix is used, so g_module_build_path returns 
"cygliblibnotify.dll", which is wrong.

--- origsrc/ario-1.3/plugins/libnotify/libnotify.ario-plugin.desktop.in	2008-12-09 12:48:26.000000000 -0600
+++ src/ario-1.3/plugins/libnotify/libnotify.ario-plugin.desktop.in	2009-07-23 02:21:52.889325800 -0500
@@ -1,5 +1,5 @@
 [Ario Plugin]
-Module=liblibnotify
+Module=libnotify
 IAge=1
 _Name=Libnotify
 _Description=Add libnotify support
